### PR TITLE
Fix logout when multiple tabs are opened

### LINKED ISSUE #558: Problem logging out when logged in in multiple tabs
Looks like the user info is being erased from the localStorage in which the log out is done but the other tab is rewriting the info into localStorage again.

### PR DESCRIPTION
**Pull request by fernandoabolafio**
closes #558

### CODE REVIEW

File: src/reducers/api.js [hunk 1]
```diff
@@ -313,6 +314,7 @@ const api = (state = DEFAULT_STATE, action) => (({
   [act.RECEIVE_LOGOUT]: () => {
     const tempState = DEFAULT_STATE;
     tempState.init = state.init;
+    clearStateLocalStorage();
```
Comment (tiagoalvesdulce) [inline src/reducers/api.js]: It's OK to do this here by now, but please write an issue to fix it in the future so our reducers stays as pure functions.

**Comment by tiagoalvesdulce** [comment]
Tested and it works great. As soon as the tests are fixed, it can go in but please note my inline comment.

File: src/reducers/api.js (hunk 1 above)
Comment (fernandoabolafio) [reply src/reducers/api.js]: Actually, there are multiple places where we need to fix that. Here is the issue:
https://github.com/decred/politeiagui/issues/562

**Comment by tiagoalvesdulce** [comment]
LGTM

**Pull request merged**